### PR TITLE
boot-qemu.sh: Show some information about the QEMU binary

### PR DESCRIPTION
**Pull request by nathanchance**
Sometimes I need to test different versions of QEMU and it is helpful to
know that I am testing the right one.

### CODE REVIEW

File: boot-qemu.sh [hunk 1]
```diff
@@ -242,7 +242,10 @@ function setup_qemu_args() {
 # Invoke QEMU
 function invoke_qemu() {
     rm -rf "${ROOTFS}"
-    zstd -d "${ROOTFS}".zst -o "${ROOTFS}"
+    zstd -q -d "${ROOTFS}".zst -o "${ROOTFS}"
+
+    printf "\n\033[01;32m%s\033[0m %s\n\n" "QEMU location:" "$(dirname "$(command -v "${QEMU[*]}")")"
+    printf "\033[01;32m%s\033[0m %s\n\n" "QEMU version:" "$("${QEMU[@]}" --version | head -n1)"
```
Comment (nickdesaulniers) [inline boot-qemu.sh:248]: can you use or define functions like `green` and `red` above? L6-12

Comment (nathanchance) [reply boot-qemu.sh]: Done, PTAL.

**Review by nickdesaulniers — APPROVED** [review]
🍫 🥢

**Review by msfjarvis — APPROVED** [review]

**Comment by nickdesaulniers** [comment]
(if we want this to show up in ci2, the submodule will have to be uprev'd)

**Comment by msfjarvis** [comment]
> (if we want this to show up in ci2, the submodule will have to be uprev'd)

Opened https://github.com/ClangBuiltLinux/continuous-integration2/pull/36

**Pull request merged**